### PR TITLE
add simplified ipv4 docker-compose files for service and checker

### PR DESCRIPTION
**Pull request by luglo**
Tested the service file via:

```
cd service
docker-compose --file docker-compose.yml.ipv4 up --build
```

And then `nc 0.0.0.0:8002` to connect to the service.

Tested the checker file via:

```
cd checker
docker-compose --file docker-compose.yml.ipv4 up --build
```

And then `http://0.0.0.0:8002` to connect to the checker.

### CODE REVIEW

**Comment by Trolldemorted** [comment]
Just replace the existing ones - I doubt anyone wants to touch that ipv6 madness ever again, and our deployment uses the compose files that are there :)

File: checker/docker-compose.yml.ipv4 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+  
+version: '3.6'
+services:
+  casino-backend:
+    build: .
+    volumes:
+      - .:/checker
```
Comment (Trolldemorted) [inline checker/docker-compose.yml.ipv4]: is this volume required? The original backend doesn't have it (?)

**Comment by DopeforHope** [comment]
> Tested the service file via:
> 
> ```
> cd service
> docker-compose --file docker-compose.yml.ipv4 up --build
> ```
> 
> And then `nc 0.0.0.0:8002` to connect to the service.
> 
> Tested the checker file via:
> 
> ```
> cd checker
> docker-compose --file docker-compose.yml.ipv4 up --build
> ```
> 
> And then `http://0.0.0.0:8002` to connect to the checker.

I'm a little bit out of the loop of the whole checker/infra stuff but why do you publish the service and checker on the same port (8002) on the host machine? Can the checker reach the service then?

File: checker/docker-compose.yml.ipv4 (hunk 1 above)
Comment (luglo) [reply checker/docker-compose.yml.ipv4]: I don't think it's required, thank you. It's removed in https://github.com/enowars/enowars3-service-casino/pull/1/commits/130ead88888bcf727e997cabe91e26e690895685.

**Comment by luglo** [comment]
> > Tested the service file via:
> > ```
> > cd service
> > docker-compose --file docker-compose.yml.ipv4 up --build
> > ```
> > 
> > 
> > And then `nc 0.0.0.0:8002` to connect to the service.
> > Tested the checker file via:
> > ```
> > cd checker
> > docker-compose --file docker-compose.yml.ipv4 up --build
> > ```
> > 
> > 
> > And then `http://0.0.0.0:8002` to connect to the checker.
> 
> I'm a little bit out of the loop of the whole checker/infra stuff but why do you publish the service and checker on the same port (8002) on the host machine? Can the checker reach the service then?

This bugged me out aswell. But the current setup is going to be deployed on two hosts one for the checker one for the service therefore the ports won't intervene each other and one has to remember only one port per service and checker.

**Comment by luglo** [comment]
Anyway there is a new issue popping up on starting the checker. My guess is it's coming from Flask/Werkzeug within the enochecker python lib (?).

```casino-backend_1  | READING INIT
casino-backend_1  | [('DEFAULT', <Section: DEFAULT>)]
casino-backend_1  | Traceback (most recent call last):
casino-backend_1  |   File "./checker.py", line 40, in <module>
casino-backend_1  |     class CasinoChecker(BaseChecker):
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/enochecker/enochecker.py", line 152, in __init__
casino-backend_1  |     cls.service = init_service(cls)  # type: Flask
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/enochecker/checkerservice.py", line 303, in init_service
casino-backend_1  |     app = Flask(__name__)
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 558, in __init__
casino-backend_1  |     self.add_url_rule(
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 66, in wrapper_func
casino-backend_1  |     return f(self, *args, **kwargs)
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1216, in add_url_rule
casino-backend_1  |     self.url_map.add(rule)
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 1386, in add
casino-backend_1  |     rule.bind(self)
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 730, in bind
casino-backend_1  |     self.compile()
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 794, in compile
casino-backend_1  |     self._build = self._compile_builder(False).__get__(self, None)
casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 945, in _compile_builder
casino-backend_1  |     code = compile(module, "<werkzeug routing>", "exec")
casino-backend_1  | TypeError: required field "type_ignores" missing from Module
```


See https://github.com/pallets/werkzeug/issues/1551 .

**Comment by luglo** [comment]
> Anyway there is a new issue popping up on starting the checker. My guess is it's coming from Flask/Werkzeug within the enochecker python lib (?).
> 
> ```
> casino-backend_1  | [('DEFAULT', <Section: DEFAULT>)]
> casino-backend_1  | Traceback (most recent call last):
> casino-backend_1  |   File "./checker.py", line 40, in <module>
> casino-backend_1  |     class CasinoChecker(BaseChecker):
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/enochecker/enochecker.py", line 152, in __init__
> casino-backend_1  |     cls.service = init_service(cls)  # type: Flask
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/enochecker/checkerservice.py", line 303, in init_service
> casino-backend_1  |     app = Flask(__name__)
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 558, in __init__
> casino-backend_1  |     self.add_url_rule(
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 66, in wrapper_func
> casino-backend_1  |     return f(self, *args, **kwargs)
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1216, in add_url_rule
> casino-backend_1  |     self.url_map.add(rule)
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 1386, in add
> casino-backend_1  |     rule.bind(self)
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 730, in bind
> casino-backend_1  |     self.compile()
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 794, in compile
> casino-backend_1  |     self._build = self._compile_builder(False).__get__(self, None)
> casino-backend_1  |   File "/usr/local/lib/python3.8/site-packages/werkzeug/routing.py", line 945, in _compile_builder
> casino-backend_1  |     code = compile(module, "<werkzeug routing>", "exec")
> casino-backend_1  | TypeError: required field "type_ignores" missing from Module
> ```
> 
> See [pallets/werkzeug#1551](https://github.com/pallets/werkzeug/issues/1551) .

We can bypass this by temporarily fixing the Python version to 3.7 instead of the actual 3.8.

**Comment by Trolldemorted** [comment]
@DopeforHope the checker and service will not run on the same host, so it should be fine

@luglo since when does werkzeug issue occur? cc @M-Munier

**Comment by DopeforHope** [comment]
> @DopeforHope the checker and service will not run on the same host, so it should be fine

I would rather have two different in case someone inexperienced wants to try this locally. But if thats the way of the current setup I will merge it.

**Pull request merged**